test/unit/effects.js

```diff
@@ -1780,7 +1781,8 @@ test("Animation callbacks (#11797)", 15, function() {
 			ok( true, "empty: start" );
 		},
 		progress: function( anim, percent ) {
-			equal( percent, 0, "empty: progress 0" );
+			equal( percent, prog, "empty: progress " + prog );
+			prog = +!percent;
```

Comment:
Just `prog = 1`, please. There's no need to be overly clever in known environments like unit tests.